### PR TITLE
1905 operation registration success

### PR DESCRIPTION
**Pull request by marcellmueller**
#1905 

### Operation Restration Success page

Very simple to test at the moment given there is nothing happening during form submission, simply navigate to the final page of the Operation Registration form and click 'Submit': 

http://localhost:3000/registration/operation/test-id/6?title=Placeholder+Title 

#### Note: the `Report a change` page doesn't exist as far as I know so it's just `tbd` currently.

Screenshot 2024-08-12 at 11 34 13 AM

### CODE REVIEW

File: bciers/apps/registration/app/components/operations/registration/FacilityInformationPage.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const FacilityInformationPage = async ({
   return (
     <FacilityInformationForm
       facilityId={facilityId}
-      isCreating={!isOperationSfo && !facilityId}
+      isCreating={!isOperationSfo || !facilityId}
```
Comment (marcellmueller) [inline bciers/apps/registration/app/components/operations/registration/FacilityInformationPage.tsx:57]: Unrelated fix I am sneaking in here. I introduced this bug which will cause an error if you pick an SFO operation on page 1, page 2 will have an error on submission if there is a facility tied to that one since it will POST instead of PUT.

File: bciers/libs/components/src/form/widgets/CheckboxWidget.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ const CheckboxWidget: React.FC<WidgetProps> = ({
           }
         />
       </div>
-      <label>{label}</label>
+      <label htmlFor={id}>{label}</label>
```
Comment (marcellmueller) [inline bciers/libs/components/src/form/widgets/CheckboxWidget.tsx:27]: This is a fix to allow clicking on the label to check the box. A nice UX improvement 😎

Comment (Sepehr-Sobhani) [reply bciers/libs/components/src/form/widgets/CheckboxWidget.tsx:27]: Ooooh I love this! I quickly attempted to do this by adding pointer-event on the whole `div` but that failed and I gave up lol. Thanks for finding the pro solution.

File: bciers/apps/registration/app/data/jsonSchema/operationRegistration/submission.ts
Comment (Sepehr-Sobhani) [inline bciers/apps/registration/app/data/jsonSchema/operationRegistration/submission.ts:1]: Thanks for fixing this!

**Review by Sepehr-Sobhani — APPROVED** [review]
Neat 🥇 !
